src/containers/NavBarScreen.tsx

```diff
@@ -18,6 +19,8 @@ interface ButtonConfig {
 }
 
 export const HeaderButton = ({ navigation, screen }: Props) => {
+	const { dispatch } = useContext(Ctx);
```

Comment:
Do we need this?

Reply:
Whoops, meant to remove that as well in the `Remove deprecated context dispatch call` commit.